test/clj/nnts2/directory/db_test.clj

```diff
@@ -0,0 +1,58 @@
+(ns nnts2.directory.db-test
+  (:require [clojure.test :refer :all]
+            [nnts2.db.directory :as directory-db]
+            [nnts2.fixtures :as fixtures]
+            [nnts2.data-factory :as factory]))
+
+(use-fixtures :each fixtures/clear)
+(use-fixtures :once fixtures/setup fixtures/adduser)
+
+(deftest create-dir-test
+  (testing "should create dir in org when given existing user, org and dir name"
+    (let [fake-dir-data  (factory/directory "fake-dir")
+          output (directory-db/create fake-dir-data)]
+      (is (not (empty? output)))
+      (is (= fake-dir-data(select-keys output [:name :parent-id :org-id :created-by-id]))))))
+
+
+(deftest create-nested-dir-test
+  (testing "should create sub directory in directory when given name, existing directory,userwith correct org"
+    (let [fake-dir-data (factory/directory "fakedir")
```

Comment:
I think we can avoid calling them `fake-` repeatedly. It's in the test namespace, so it's mostly apparent.

Reply:
yes, will do. it was annoying me also. But I wrote it once so kept doing that. Will change